.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish to testpypi repo
+
+on: push
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to TestPyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Run build script
+      run: >-
+        bash build_release.sh
```

Comment:
If the tests fail here, does it prevent it from publishing (that is the desired behavior)